src/test/java/uk/co/ramp/covid/simulation/output/DailyStatsTest.java

```diff
@@ -137,6 +148,11 @@ public void testDailyStats() {
         assertEquals("Inconsistent number of deaths", expDeaths, stats.get(0).get(nDays - 1).getDead());
         assertEquals("Inconsistent number of infected", expInfected, dailyInfected);
         assertEquals("Inconsistent number of place infections", expPlaceInfections, dailyInfected);
+        
+        assertNotEquals("Some people are hospitalised", hospitalised);
+        assertNotEquals("Some people are hospitalised", newlyHospitalised);
+        assertTrue("Hospitalised should be > newlyHospitalised", newlyHospitalised > 0);
```

Comment:
I don't understand these asserts.  Are these calls to assertNotEquals() comparing strings to ints?  And shouldn't the assertion with a comment about "Hospitalised should be > newlyHospitalised" be making a corresponding comparison?

Reply:
Thanks for spotting this, looks like somethings strange has gone on here. I'll fix this in a new PR